.github/ISSUE_TEMPLATE/1-bug.yml

```
@@ -0,0 +1,145 @@
+name: 🐛 Bug report
+description: Describe a problem
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Some general tips:
+
+        - Is this really a problem?
+        - Is this a problem here?
+        - Can this be solved in a different way?
+
+  - type: checkboxes
+    id: initial-checklist
+    attributes:
+      label: Initial checklist
+      options:
+        - label: I read the [support docs](https://mdxjs.com/support)
+          required: true
+        - label: I read the [contributing guide](https://mdxjs.com/contributing)
+          required: true
+        - label: I agree to follow the [code of conduct](https://github.com/mdx-js/.github/blob/HEAD/code-of-conduct.md)
+          required: true
+        - label: I searched [issues](https://github.com/search?q=user%3Amdx-js&type=Issues) and couldn’t find anything (or linked relevant results below)
+          required: true
+
+  - type: input
+    id: affected-versions
+    attributes:
+      label: Affected packages and versions
+      description: Please test using the latest version of the relevant packages to make sure your issue has not already been fixed.
+    validations:
+      required: true
+
+  - type: input
+    id: runnable-example
+    attributes:
+      label: Link to runnable example
+      description: |
+        Link to repository or sandbox with runnable example of the issue.
+
+        Some CodeSandbox starters:
+        - [MDX Evaluate with Create React App](https://codesandbox.io/s/mdx-evaluate-create-react-app-kg630)
+        - [MDX Loader with Create React App](https://codesandbox.io/s/mdx-loader-create-react-app-9e39f)
+
+        Alternatively, use the next section *Steps to reproduce*.
+    validations:
+      required: false
+
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: How did this happen? Please provide a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: What should happen?
+    validations:
+      required: true
+
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual behavior
+      description: What happens instead?
+    validations:
+      required: true
+
+  - type: dropdown
+    id: javascript-runtime
+    attributes:
+      label: Runtime
+      description: |
+        What JavaScript runtime are you using to build or run the code?
+        **Note**: for ECMASCript modules (ESM) in Node, at minimum Node.js 12.20, 14.14, or 16.0 is required.
+      multiple: true
+      options:
+        - Node v17
+        - Node v16
+        - Node v14
+        - Node v12
+        - Deno v1
+        - Electron v15
+        - Electron v14
+        - Electron v15
```

```suggestion
        - Deno
        - Electron
```

I haven’t ever heard of an Electron version being of importance to MDX. Neither are we developing for it or tracking it. So it’s really quickly getting out of date (see remarkjs/.github).